app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventFragment.java

```diff
@@ -0,0 +1,64 @@
+package de.tum.in.tumcampusapp.component.ui.ticket;
+
+import android.annotation.SuppressLint;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import java.util.List;
+
+import de.tum.in.tumcampusapp.R;
+import de.tum.in.tumcampusapp.component.ui.ticket.model.Event;
+
+/**
+ * Created by SONU on 16/09/15.
+ */
```

Comment:
please :pray: go through the code at least once when you copy :spaghetti: it to remove unnnecessary comments

Reply:
Thank you. Fixed.